packages/k6-tests/src/sample/kitchen-sink.ts

```diff
@@ -89,17 +89,20 @@ export default function ({ userInfos, adminCredential }: Data): void {
   const folderMovedName = randomString();
   const assetName = randomString();
   userClient.resource.create(userHome, folderCreationName);
-  defer.push(() => userClient.resource.delete(userHome, folderMovedName));
+  defer.push(() => {
+    return userClient.resource.delete(userHome, folderMovedName)
+  });
   userClient.resource.move(userHome, folderCreationName, folderMovedName);
   userClient.resource.upload(userHome, [folderMovedName, assetName].join('/'), randomBytes(1000));
   userClient.resource.download(userHome, [folderMovedName, assetName].join('/'));
 
-  const createdShareResponse = userClient.share.create(
-    folderMovedName,
-    adminCredential.login,
+  const shareeSearchResponse = userClient.search.sharee(adminCredential.login, ItemType.folder)
```

Comment:
then we can `userClient.sharee.get(adminCredential.login, ItemType.folder)`

Reply:
see above, sorry 😢